pages/css/css.md

```diff
@@ -5,6 +5,4 @@ sidenav: css
 ---
 
 # CSS
-The purpose of the CSS coding styleguide is to create consistent CSS or
-preprocessor CSS code across 18F. The styleguide should be treated as a guide
-&mdash; rules can be modified according to project needs.
+The purpose of the CSS coding styleguide is to help 18F developers make decisions about CSS tools and implementations. The goal of this guide is not to be dogmatic, but to explain how and why we recommend certain approaches, and to help developers make their own decisions for their individual projects.
```

Comment:
The guide hasn't created consistency, so it seems a bit pompous to suggest that that's the goal.

Reply:
I think we _should_ be dogmatic, where being dogmatic is warranted. We should just say why we care.

For example, at this point, I'm fairly comfortable saying atomic CSS should not be used for long-lived code unless the partner is already using it and you're adopting their house style. It's just too difficult to pick through when you want to make changes, and unfamiliar to too many partners.